accounts/abi/bind/backends/simulated.go

```diff
@@ -712,6 +712,18 @@ func (fb *filterBackend) ServiceFilter(ctx context.Context, ms *bloombits.Matche
 	panic("not supported")
 }
 
+func (fb *filterBackend) AccountExtraDataStateGetterByNumber(context.Context, rpc.BlockNumber) (vm.AccountExtraDataStateGetter, error) {
```

Comment:
Why did you have to add this block? (it looks like it shoud have already been there)

Reply:
I didn't add it. 
What you are looking is specifically the commit that did the merge from `master` into this branch after the multi-tenancy was merged. Therefore, on that commit, you will see that it was added.

However, if you check the latest version of the file in the PR (as this comment is marked on an outdated file), you see that it wasn't added on this PR.